### ISSUE TITLE
untag

### ISSUE THREAD

**Issue by arademaker** [own-pt/sensetion.el#185]
It would be good to have a command for untag, remove any sense annotation for a glob or token. I remember that we have a command to show a menu with all commands, but I could not find in the wiki references to it.

**Comment by odanoburu** [comment]
you would like a command to clear all the annotations? we refrained from having this because usually its not much more labor to untag each assigned sense individually, specially if the clearing command prompts the user for confirmation (as it probably should, since it may be pressed unwillingly). personally I think it's good to have the user think hard about such a decision that may erase data, having to look at each annotation to remove it seems good to me.

but such a command could be added, of course. it would be a matter of creating a function similar to `sensetion-edit-unsure` (at `sensetion-edit.el`) and maybe giving it a keybinding.

> I remember that we have a command to show a menu with all commands, but I could not find in the wiki references to it.

the `s` command. I think the command table was lost at some point (it can be reconstructed from the `s` command and added to the Usage section of the wiki)

**Comment by arademaker** [comment]
But we can't untag the last sense, right?  OK, added in the wiki the info about `s`.

**Comment by odanoburu** [comment]
not exactly, no. I could implement a pop-sense command that removes the last-added sense. would you that work for you? I don't think it's a good idea because there's already a reasonably fast way of removing a sense; for this to be faster it would have to not ask for user confirmation, which might increase accidental data loss . if it'd ask for user confirmation, it would hardly be more efficient than the current method…

**Comment by arademaker** [comment]
not clear for me. What is pop-sense? I don't like the idea of adding complexity. Last added sense means we would have to control order of changes. What is the currently reasonably fast way for removing saense? If the glob/word has one sense annotated? 

Meanwhile, I can use the `unsure` mark to no lose the track of things that I need to revise. Les us keep this open until we don't have a clear idea.

**Comment by odanoburu** [comment]
> What is pop-sense? 

senses are stored in order of addition (more recent first in the list). the command would remove the first element at each call.

> What is the currently reasonably fast way for removing sense? If the glob/word has one sense annotated?

the one you have in mind, which is the same regardless of the number of senses: bring up the sense menu and unselect the senses you want to remove.
